### PR TITLE
Add `.message--success` variant for happy times

### PR DESCRIPTION
**Pull request by tdooner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added `.message--success` for those times when things just... go well.
```
Comment (fritzjooste) [inline CHANGELOG.md]: :joy: i love this

File: _sass/molecules/_message.scss [hunk 2]
```diff
@@ -21,6 +21,16 @@
   padding-right: 0.5em;
 }
 
+.message--success {
+  background-color: $color-green;
+
```
Comment (fritzjooste) [inline _sass/molecules/_message.scss]: nit: is this extra return space necessary here?

File: _molecules/messages.md [hunk 3]
```diff
@@ -27,6 +27,19 @@ order: 3
...
+	<div class="message message--success">
+		<div class="message__icon">
+			<i class="fa fa-check-circle"></i>
+		</div>
+		<div class="message__content">
+			<strong>Great!</strong> Everything went well with your action. <a href="#">Visit your new object.</a>
+		</div>
+	</div>
+</div>
+
```
Comment (fritzjooste) [inline _molecules/messages.md]: We should convert to indentation with two spaces (instead of tabs) at some point, for consistency.

**Comment by tdooner** [comment]
Merged in https://github.com/codeforamerica/style-guide/commit/e670a9fd0740bc5a6251e1f4a1087393e7e3589c and released in v4.0.9

**Pull request closed without merging**